src/main/java/penguin/storage/Storage.java

```diff
@@ -0,0 +1,78 @@
+package penguin.storage;
+
+import penguin.exception.PenguinException;
+import penguin.task.TaskList;
+import penguin.task.Task;
+import penguin.task.TaskCode;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.io.FileWriter;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Scanner;
```

Comment:
Just a small nitpick but perhaps you could organise by package
 
```suggestion
import java.io.File;
import java.io.IOException;
import java.io.FileWriter;
import java.nio.file.Files;
import java.nio.file.Path;
import java.nio.file.Paths;
import java.util.Scanner;
```